js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
No, but the user might be going to other websites that store more data, and data is cleared on an LRU policy, not a "biggest users cleared first" policy. There's a reason most users aren't affected, only a small number of users.

And this is, of course, much more of a problem on mobile.

Hm, I think overall this is probably not going to be a huge problem. At some point, I want to move to server-side database, anyway.